package.json

```diff
@@ -15,7 +15,12 @@
     "test": "c8 tap",
     "test:ci": "npm run lint && npm run build && npm run test:coverage",
     "test:coverage": "c8 --reporter=lcov tap --cov",
-    "prepack": "npm run build"
+    "prepack": "npm run build",
+    "bench": "npm run benchmark:piscina-default && npm run benchmark:piscina-fixed-queue && npm run benchmark:queue-comparison && npm run benchmark:piscina-comparison",
+    "benchmark:piscina-default": "node benchmark/simple-benchmark.js",
+    "benchmark:piscina-fixed-queue": "node benchmark/simple-benchmark-fixed-queue.js",
+    "benchmark:queue-comparison": "node benchmark/queue-comparison.js",
+    "benchmark:piscina-comparison": "node benchmark/piscina-queue-comparison.js"
```

Comment:
Let's maybe split these and aggregate them into two different scopes `bench:taskqueue` and we attempt to run all the TaskQueue related benchmarks, and left `bench` to run all benchmarks.

We might add more with the work over the new load balancer and scheduler that I'd like to work on soon.

Reply:
Good point

Reply:
Done